### PR TITLE
Create sprint-98-summary.md

### PR DESCRIPTION
**Pull request by victoriaatraft**
Sprint 98 summary details

### CODE REVIEW

**Review by reitermb — APPROVED** [review]

**Review by andrew-jameson — APPROVED** [review]

File: docs/Sprint-Review/sprint-98-summary.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  * #2795 [Spike] - As tech lead, I need TDP to detect duplicate records within a file and not store them in the db
+
+* DevOps: *Successful deployments across environments and pipeline stability investments*
+    * #2870 ES re-indexing automation
+
+* Design: Close out all in-progress/in-review work and begin refinement to support upcoming sprints/roadmap items*
+  * #2847 Knowledge Center Content
+  * #2473 Email notification template supporting #2473 (due date reminders)
+  * #2801 Friendly Name changes
+  * #2909 [Spike] research spike on-deck pending close-out of the above
```
Comment (lfrohlich) [inline docs/Sprint-Review/sprint-98-summary.md]:
```suggestion
  * #2909 [Spike] Out of the box OFA Kibana Experience & DIGIT Data Access (pending close-out of the above)
```

**Review by lfrohlich — CHANGES_REQUESTED** [review]
Will approve after minor change

**Comment by victoriaatraft** [comment]
> Will approve after minor change

suggestions committed.

**Review by lfrohlich — APPROVED** [review]
LGTM

**Pull request merged**